Source/Core/UICommon/GameFileCache.cpp

```diff
@@ -58,8 +58,9 @@ size_t GameFileCache::GetSize() const
   return m_cached_files.size();
 }
 
-void GameFileCache::Clear()
+void GameFileCache::Clear(bool delete_on_disk)
 {
+  File::Delete(m_path);
```

Comment:
I'm guessing there is an `if (delete_on_disk)` missing here?

Reply:
Oops, yes :(